public/application.js

```diff
@@ -133,9 +146,17 @@ function initialize() {
       }
     },
     processPastMeetups: function () {
-      const now = new Date()
+      const now = moment()
+      const endOfDay = moment().endOf('day')
+
       this.pastMeetups = this.meetups
-        .filter((meetup) => new Date(meetup.datetime) < now)
+        .filter((meetup) => {
+          const meetupDate = moment(meetup.datetime)
+          return (
+            meetupDate.isBefore(now, 'day') ||
+            (meetupDate.isSame(now, 'day') && now.isAfter(endOfDay))
```

Comment:
Maybe like that ?

```suggestion
            now.isSameOrBefore(meetupDate, 'day')
```

Reply:
After adding this, I don't see previous meetups

Reply:
After speaking with @dawidrudnik we fixed this